sources/gov/kaunas/municipality/educational_institutions.yaml

```diff
@@ -0,0 +1,30 @@
+id: "gov/kaunas/municipality/educational_institutions"
+title: "Educational institutions"
+description: "CSV file containing data about Kaunas educational institutions"
+type: "source"
+source:
+  - "https://github.com/Kauno-miestas/Svietimas/blob/master/Svietimo%20istaigos.csv"
+  - "csv:"
```

Comment:
There is no need for `source` here, because this source is about single object, not about multiple objects.

Reply:
Ok, found what is the issue. File should be named `.yml`, not `.yaml`. Please rename it, and you will get automated checks.

Reply:
"There is no need for source here," I dont understand how it should look like